config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
   resources :chips, only: [:index]
   resources :oil_types, only: [:index, :show], param: :name
+  get '/:name', to: redirect('/oil_types/%{name}'), as: "oil_name"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.